serpent-os/recipes
# bash: Enable support for sourcing from ~/.bashrc.d OOTB

**Test Plan**

- Ensure that the user shell in `/etc/passwd` is `/usr/bin/bash`
- Remove any pre-existing `~/.bash{rc,_profile}`
- `mkdir -pv ~/.bashrc.d`
- `echo 'export KILLROY_WAS_HERE=bash' > ~/.bashrc.d/99_KILLROY_WAS_HERE.bash`
- Open new terminal
- Check that  `echo $KILLROY_WAS_HERE` outputs `bash`
